dotcom-rendering/src/components/Lightbox.stories.tsx

```diff
@@ -149,33 +151,6 @@ export const WithEverything = () => {
 	);
 };
 
-export const WithoutCaption = () => {
```

Comment:
Now we are testing interactions elsewhere, it doesn't make sense to test the information column without any caption, as this is unlikely and undesirable behaviour to see on the site

Reply:
Yes, and we have [an integration test to ensure that the preferences are saved](https://github.com/guardian/dotcom-rendering/blob/53b2ed4eb80a303c8504eb8f93db6934c86a0043/dotcom-rendering/playwright/tests/lightbox.e2e.spec.ts#L283-L328)!